consensus/istanbul/core/roundchange_v2.go

```diff
@@ -238,7 +223,7 @@ func (c *core) handleRoundChangeV2(msg *istanbul.Message) error {
 // ----------------------------------------------------------------------------
 
 // CurrentRoundChangeSet returns the current round change set summary.
-func (c *core) CurrentRoundChangeSetV2() *RoundChangeSetSummary {
+func (c *core) CurrentRoundChangeSet() *RoundChangeSetSummary {
```

Comment:
You've been maintaining the V2 suffix in all the PR except this one. Just checking if that was on purpose or not